paper/paper.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Streamz: A small library to manage continuous streams of data'
+tags:
+  - Stream
+  - Dask
+authors:
+ - name: Matthew Rocklin
+   orcid: ?
+   affiliation: 1
+ - name: Christopher J. Wright
+   orcid: 0000-0003-2522-7028
+   affiliation: 2
+ - name: Julien Lhermitte
+   orcid: 0000-0003-0660-975X
+   affiliation: 5
+ - name: Daniel B. Allan
+   orcid: ?
+   affiliation: 4
+ - name: Kevin G. Yager
+   orcid: 0000-0001-7745-2513
+   affiliation: 5
+ - name: Simon J. L. Billinge
+   orcid: 0000-0002-9734-4998
+   affiliation: 2, 3
+affiliations:
+ - name: Continuum Analytics
+   index: 1
+ - name: Department of Applied Physics and Applied Mathematics, Columbia University
+   index: 2
+ - name: Condensed Matter Physics and Materials Science Department, Brookhaven National Laboratory
+   index: 3
+ - name: NSLS-II, Brookhaven National Laboratory
```

Comment:
@danielballan should this be spelled out in full or abbreviated?

Reply:
I think NSLS-II works fine here, given that Brookhaven is spelled out.